media/css/foundation/annual-report-2019.scss

```diff
@@ -534,14 +551,18 @@ $image-path: '/media/protocol/img';
 
 .c-leadership-article {
     h3 {
-        @include text-title-sm;
-        border-left: 3px solid $color-black;
-        padding: $spacing-md 0 $spacing-md $spacing-lg;
-
+        @include text-title-md;
+        // border-left: 3px solid $color-black;
+        padding: $spacing-lg 0 $spacing-sm;
+        // margin-top: $spacing-2xl;
```

Comment:
If these are being removed we should just remove them instead of commenting

Reply:
🙈